packages/nns/src/governance.canister.ts

```diff
@@ -462,6 +464,41 @@ export class GovernanceCanister {
     });
   };
 
+  /**
+   * Simulate merging two neurons
+   *
+   * @throws {@link GovernanceError}
+   */
+  public simulateMergeNeurons = async (request: {
+    sourceNeuronId: NeuronId;
+    targetNeuronId: NeuronId;
+  }): Promise<NeuronInfo> => {
+    const rawRequest = toMergeRequest(request);
+
+    const command = await simulateManageNeuron({
+      request: rawRequest,
+      service: this.certifiedService,
+    });
+
+    if (
+      "Merge" in command &&
+      command.Merge.target_neuron_info[0] !== undefined &&
+      command.Merge.target_neuron[0]?.id[0]?.id !== undefined
+    ) {
+      return toNeuronInfo({
+        neuronId: command.Merge.target_neuron[0].id[0].id,
+        neuronInfo: command.Merge.target_neuron_info[0],
+        rawNeuron: command.Merge.target_neuron[0],
+        canisterId: this.canisterId,
+      });
+    }
+
+    // Edge case
+    throw new UnrecognizedTypeError(
+      `Unrecognized Merge error in ${JSON.stringify(command)}`
```

Comment:
Maybe add the word "simulated" in the error msg?

Reply:
`Merge` refers to the expected result type.
I've added the function name `simulateMergeNeurons` in the message now, although I guess it will be in the stack trace as well.